src/components/layouts/auth-wrapper/auth-wrapper.js

```diff
@@ -73,6 +87,29 @@ class AuthWrapper extends React.PureComponent {
     shouldLogout: false
   };
 
+  componentDidMount() {
+    let token =
+      parse(this.props.location.search) &&
+      parse(this.props.location.search).token
+        ? parse(this.props.location.search).token
+        : undefined;
+
+    if (token) {
+      // eslint-disable-next-line no-undef
+      localStorage.setItem("token", token);
+
+      // eslint-disable-next-line no-undef
+      const lastUrlBeforeLogin = window.localStorage.getItem("last-visited");
+      if (lastUrlBeforeLogin) {
+        // eslint-disable-next-line no-undef
+        localStorage.removeItem("last-visited");
+        navigate(`/${lastUrlBeforeLogin}`);
+      } else {
+        navigate("/");
```

Comment:
This shouldn't navigate to "/" but to the current url without the token as a query param

Reply:
This acts as fall back. 

I assume that the only way to login is to use a `login` link, redirect to `https://auth.mozilla.auth0.com` and then back to the OSSN page.

So for every login action, the `last-visited` key at the local storage will be set and used to redirect the user back.

This fallback is being used in case that there is a token (and thus a login action), but the local storage does not have the `last-visited` key set.

Reply:
I know it's a fallback, but the fallback should redirect to user's current page not to the homepage by default. Otherwise when the backend redirects to `/member/:id?edit=true&initial=true&token=xxx` the member won't be redirected to the edit page